evals/compute_scores.py

```diff
@@ -335,9 +335,16 @@ def main():
                 hypo_texts = []
                 ref_texts = []
                 for k in seg2text.keys():
-                    hypo_texts.append(seg2text[k])
-                    ref_texts.append(refs_map[k][0])
-                    src_texts.append(srcs_map[k])
+                    if k.split("#")[1] == "_testsuite_":
+                        continue
```

Comment:
@thammegowda @snukky  I had to filter out testsuite outputs as we do not have references for them. Shouldn't be a problem to cause a mismatch from what is generated and what is evaluated, but just fyi in case you can think of possible issues I couldn't think of.